src/main/java/uk/gov/justice/hmpps/prison/service/v1/NomisApiV1Service.java

```diff
@@ -258,7 +258,7 @@ private String convertToJsonString(final EventSP e) {
     public Image getOffenderImage(final String nomsId) {
         final var imageBytes = offenderV1Repository.getPhoto(nomsId).orElseThrow(EntityNotFoundException.withId(nomsId));
 
-        return Image.builder().image(DatatypeConverter.printBase64Binary(imageBytes)).build();
+        return Image.builder().image(Base64.getEncoder().encodeToString(imageBytes)).build();
```

Comment:
googling found https://stackoverflow.com/questions/38563609/datatypeconverter-vs-base64 and to use `Base64` instead